charts/astronomer/templates/ingress.yaml

```diff
@@ -34,7 +34,9 @@ spec:
       hosts:
         - {{ .Values.global.baseDomain }}
         - app.{{ .Values.global.baseDomain }}
+      {{- if eq .Values.global.plane.mode "unified" }}
```

Comment:
lets use variable and call in rest of the places this will reduce conditional changes in multiple places